pallets/loans/src/lib.rs

```diff
@@ -629,6 +629,30 @@ pub mod pallet {
             Ok(().into())
         }
 
+        /// Sets a new liquidation incentive percentage for `currency_id`.
+        ///
+        /// Does nothing if the provided asset is not attached to an existent incentive.
+        ///
+        /// - `currency_id`: the asset that is going to be modified.
+        #[pallet::weight(T::WeightInfo::set_liquidation_incentive())]
+        #[transactional]
+        pub fn set_liquidation_incentive(
+            origin: OriginFor<T>,
+            currency_id: CurrencyId,
+            liquidate_incentive: Rate,
+        ) -> DispatchResultWithPostInfo {
+            T::UpdateOrigin::ensure_origin(origin)?;
+            Self::ensure_currency(&currency_id)?;
+
+            if <LiquidationIncentive<T>>::try_get(currency_id).is_err() {
+                return Err(<Error<T>>::CurrencyNotEnabled.into());
+            }
+
+            <LiquidationIncentive<T>>::insert(currency_id, liquidate_incentive);
+
+            Ok(Some(T::WeightInfo::set_liquidation_incentive()).into())
```

Comment:
There is no need to do this, based on the doc
```
/// The return typ of a `Dispatchable` in frame. When returned explicitly from
/// a dispatchable function it allows overriding the default `PostDispatchInfo`
/// returned from a dispatch.
```

Reply:
Nice! Thank you